_data/cryptocurrencies.yml

```diff
@@ -379,6 +379,13 @@ websites:
       - totp
     doc: https://support.poloniex.circle.com/hc/en-us/articles/360025399352
 
+  - name: Remitano
+    url: https://remitano.com/
+    img: remitano.png
+    tfa:
+      - totp
```

Comment:
```suggestion
      - proprietary
```
Currently, when the site uses "Authy" it's `proprietary` instead of `totp`.

Reply:
We used totp not Authy. Because Authy has feature to resync from cloud, so we just recommend customers to use with Authy, but the customer can freely use with Google Authenticator

Reply:
Okay, what's strange to me is the fact that the code in the screenshots has 7 digits and 20 seconds lifetime. That lead me to believe that Google Auth would be lost. Is the screenshot just from another process?

Reply:
Here is the screenshot of enabling two factor process: 
![Settings](https://user-images.githubusercontent.com/344684/69321610-280cb700-0c8f-11ea-8f67-cd3219d43b57.jpg)

User can definitely scan the QR or input manually the code via Google Authenticator.

Reply:
That code lists your username.  I verified with google authenticator using the code above.  You probably should remove that for privacy reasons.

Reply:
Alright, seems to work fine in GA (1P). Will resolve this conversation.

Reply:
@tamvm you should delete that comment, even though it has been edited, it still shows the original message.

Reply:
Thanks @Rfosu2k2.